purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -213,8 +213,8 @@ class Purchases internal constructor(
 
     /**
      * This method will send all the purchases to the RevenueCat backend. Call this when using your own implementation
-     * for subscriptions anytime a sync is needed, like after a successful purchase, or when migrating existing
-     * users to RevenueCat
+     * for subscriptions anytime a sync is needed, such as when migrating existing users to RevenueCat
+     * 
```

Comment:
<img width="1182" alt="image" src="https://user-images.githubusercontent.com/3922667/215510755-b5aa7ff9-bfd1-47b7-811c-465aa6d9abe5.png">

Not sure what's going on with the thing it's complaining about from line 1, but it's safe to ignore since it wasn't modified in this PR. 
The other one, however, is from this PR, and it's complaining about this extra line (217), since we'd have two consecutive lines that have nothing in it. 
Could you remove line 217?